package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "x-common",
+  "name": "problem-specifications",
```

Comment:
Does this change have any ramifications? I'm unfamiliar with whatever consumes this JSON file.